schema/zk-extensions.asn

```diff
@@ -1,35 +0,0 @@
-Zk-Extensions
```

Comment:
I've created a zk-extensions branch that we can use to easily find this file again, if we want it in the future for some reason.